.github/workflows/docker_build_deploy.yml

```diff
@@ -31,6 +31,9 @@ jobs:
           echo "{\"commit\":\"$GITHUB_SHA\",\"version\":\"$GITHUB_REF\",\"source\":\"https://github.com/$GITHUB_REPOSITORY\",\"build\":\"$GITHUB_RUN_ID\"}" > version.json
 
       - name: Build Docker image
+        env:
+          UPLOAD_SENTRY_SOURCEMAPS: true
```

Comment:
Looks like we no longer need the env var `UPLOAD_SENTRY_SOURCEMAPS`?

I believe we can remove it from 
- `.env-dist`
- `next.config.js`

Reply:
I think it still needs to be set as a variable, we need some way to get it into `next.config.js` right? I'm just setting it in the yaml vs. setting it in the GHA environment and then passing it through the yaml.

We only want it to run in this context, not on every PR and every time someone might run `npm run build` locally right?

Reply:
Ah hmm, I think I misunderstood what's going on here. NVM!

Reply:
np, thanks!